model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -40,6 +40,18 @@ object Attributable {
     map.values.map(countAttributes).sum
   }
 
+  def safePrint(map: AttributeMap, depth: Int = 10): String = {
+    def safePrintInner(attr: Attribute): String = {
+      attr match {
+        case attr: AttributeListElementable => attr.toString
+        case attrList: AttributeList[_] => attrList.list.take(depth).toString
+      }
+    }
+
+    val keys = map.keys.take(depth)
+    val values = map.values.take(depth).map(safePrintInner)
```

Comment:
This seems a little confusing - so a depth of 10 would actually print "the first 10 values for each of the first 10 keys"?

Reply:
Yeah, basically I'm trying to eliminate the possibility that any dimension of nesting has a high number of values. It was definitely challenging finding a way to do this that was both informative _and_ wouldn't hog CPU concatenating strings. Open to suggestions.

Reply:
Short of renaming it to `depthAndWidth` I don't think there's much to do here. Maybe a comment on how to use the parameter? Or maybe it's simple enough that people should be able to work it out on their own?